packages/fxa-auth-server/scripts/must-reset.js

```diff
@@ -20,68 +20,64 @@
 // HACK: Prevent config falling over due to missing secrets
 process.env.NODE_ENV = 'dev';
 
-const butil = require('../lib/crypto/butil');
 const commandLineOptions = require('commander');
-const config = require('../config').getProperties();
-const crypto = require('crypto');
-const log = require('../lib/log')({});
-const P = require('../lib/promise');
+const fs = require('fs');
+const main = require('./must-reset/index');
 const path = require('path');
-const Token = require('../lib/tokens')(log, config);
-const AuthDB = require('../lib/db')(config, log, Token);
-const oauth = require('../lib/oauth/db');
 
 commandLineOptions
-  .option('-i, --input <filename>', 'JSON input file')
+  .option('-e, --emails [emails]', 'Email addresses')
+  .option('-u, --uids [uids]', 'User IDs')
+  .option('-i, --input <filename>', 'Input filename from which to read input')
   .parse(process.argv);
 
-const requiredOptions = ['input'];
+if (!commandLineOptions.input) {
+  console.error(`-i, --input required`);
+  process.exit(1);
+} else if (!commandLineOptions.emails && !commandLineOptions.uids) {
+  console.error('One of `emails` or `uids` must be specified');
+  process.exit(1);
+} else if (commandLineOptions.emails && commandLineOptions.uids) {
+  console.error('Only one of `emails` or `uids` can be specified, not both');
+  process.exit(1);
+}
+
+let emails = [];
+let uids = [];
+
+if (commandLineOptions.emails) {
+  emails = getItems('emails');
+} else if (commandLineOptions.uids) {
+  uids = getItems('uids');
+}
 
-function checkRequiredOption(optionName) {
-  if (!commandLineOptions[optionName]) {
-    console.error(`--${optionName} required`);
+if (!emails.length && !uids.length) {
+  console.error('No `emails` or `uids` found inside the specified file');
+  process.exit(1);
+}
+
+function getItems(type) {
+  let input = '';
+  try {
+    input = fs
+      .readFileSync(path.resolve(commandLineOptions.input))
+      .toString('utf8');
```

Comment:
If the return statement from below was added after this, the input could be a const.

Reply:
changed